horizons/world/player.py

```diff
@@ -69,7 +69,8 @@ def initialize(self, inventory):
 
 	def __init(self, name, color, clientid, difficulty_level, max_tier_notification, settlerlevel=0):
 		assert isinstance(color, Color)
-		assert isinstance(name, basestring) and name
+		assert isinstance(name, basestring)
+		assert name
```

Comment:
I think we should leave them as they are and discuss in general how to proceed with asserts. I've heard approval of @ChrisOelmueller on IRC to remote isinstance checks already, maybe we should drop type-checks alltogether.

With the move to python3 we *could* make use of [PEP-484](https://www.python.org/dev/peps/pep-0484/) and/or [mypy](http://mypy-lang.org/index.html) or similar.